portal/src/components/roles-and-groups/list/UserGroupsList.tsx

```diff
@@ -141,6 +161,22 @@ export const UserGroupsList: React.VFC<UserGroupsListProps> =
               />
             );
           }
+          case UserGroupsListColumnKey.Role: {
+            const text =
+              item.roles.totalCount === 0
+                ? "-"
+                : item.roles.items
+                    ?.slice(0, 3)
+                    .map((item) => item.name)
+                    .join(", ");
+            const addtionalInfo =
+              item.roles.totalCount > 3 ? ` +${item.roles.totalCount - 3}` : "";
+            return (
+              <BaseCell>
+                <Text className="whitespace-normal text-inherit">{`${text}${addtionalInfo}`}</Text>
```

Comment:
`whitespace: normal` seems to be the default, why do we need it here?
And I think this cell should have same text style as other cells (TextCell), why we need `text-inherit`?

Reply:
Seems the detailed list has injected styles into cell to make it not wrap.
For the color part, actually it's other cells not using `Text` but `div` for wrapping the text, let me update it.

Reply:
It seems we should not do wrapping in detailed list. If there is a very long role name, will it break the layout?

Reply:
<img width="804" alt="image" src="https://github.com/authgear/authgear-server/assets/63200569/cddf782e-7e01-485e-9b19-c9551fc28c17">

It will just make the row taller which look fine to me

Reply:
But it is unlimited, that means, if the text is long enough it can occupy the whole screen.
I also found the same problem in the description cell of roles and groups.
Please use line clamp to limit the maximum line to 2:
https://tailwindcss.com/docs/line-clamp#truncating-multi-line-text

Reply:
Then seems like `+5` at last would seldom been displayed

But in terms of UX, I think user want to see something like `Human Resource, ..., +5` rather than `Human Resource , Next...`?

Reply:
You are right. What about something like this?
```
<p class="flex">
  <span class="line-clamp-2">Human Resource Manager, IT Admin</span>
  <span class="flex items-end">+5</span>
</p>
```

Reply:
It looks ok when the line is clamped but strange if it's not. We can't determine how to position this `+5` 

<img width="222" alt="image" src="https://github.com/authgear/authgear-server/assets/63200569/e451dfe4-be8e-402b-b53c-b4c1c058c7b3">


<img width="213" alt="image" src="https://github.com/authgear/authgear-server/assets/63200569/caa6515a-1ec4-43c5-a67c-eb3b8778a689">

Reply:
Maybe I just use `line-clamp-2` first.
See if @fungc-io is ok with chop off the `+5` at last?

Reply:
The "+5" is better following the last role name instead of showing on the side. Is that very hard to do?

Reply:
I think a simple approach would be to display the +X at a new line. i.e.

```
Human Resource Manager, HR
Admin, Some Admin
+3
```
```
Human Resource Manager, HR
Admin, Some Admin, Some other...
+3
```

Reply:
Maybe add a tooltip instead of +3?